HNNMain.js

```diff
@@ -1,8 +1,15 @@
 import React from 'react';
+import HNNAppBar from "./component/settings/HNNAppBar";
 
-export default class HNNMain extends React.Component {
-
-    render() {
-        return (<h1>Hello world!</h1>)
-    }
-}
+export default ({ data }) => {
+	if (data == undefined) {
+		return <div />
+	}
+	else {
+		window.isDocker = data.isDocker;
```

Comment:
@rodriguez-facundo let's use context instead https://reactjs.org/docs/context.html for all of them.